adhocracy4/polls/exports.py

```diff
@@ -106,14 +121,24 @@ def get_field_data(self, item, field):
         index, user = item
 
         if field == "user_id":
-            value = index + 1
-
+            # For anonymous Users Show Anon + count for registered show the count
+            if hasattr(user, "pk"):
+                value = index + 1
+            else:
+                value = "Anon" + str(index + 1)
```

Comment:
@t-j-q for the field of the users, when there are unregistered, we don't want to use their long UUID number, so is it fine to have the prefix "ANON"?